packages/vm/lib/evm/interpreter.ts

```diff
@@ -1,5 +1,4 @@
-import BN = require('bn.js')
-import Account from '@ethereumjs/account'
+import { Account, BN } from 'ethereumjs-util'
```

Comment:
There is still a reference to `ethereumjs-account` left in the code docs of this file (on the `step` event).

Side note: does the `step` event also need some attention on this regard? 🤔  At least a mention in the release notes?

Reply:
updated in https://github.com/ethereumjs/ethereumjs-vm/pull/911/commits/12ff41650e8add0ceaba1f6b48e43d0b6200d88a, yes I think we should mention in the release notes.